docs/add-ins/manifests.md

```diff
@@ -367,6 +367,16 @@ The following example specifies a domain  `https://www.contoso2.com` as a second
 
 App domains are also necessary to enable cookie sharing between the pop-out window and the add-in running in the rich client.
 
+The following table describes the browser behavior for URLs not belonging to the add-in's default domain.
+
+|Outlook client|Domain defined<br>in AppDomains?|Browser behavior|
+|---|---|---|
+|All clients|Yes|Link opens in add-in task pane.|
+|Outlook 2013 on Windows<br>Outlook 2016 on Windows (one-time purchase)|No|Link opens in Internet Explorer 11.|
```

Comment:
Is the text "(one-time purchase)" intended to apply to both "Outlook 2013 on Windows" and "Outlook 2016 on Windows" ?  If yes, then I'd suggest adding the text "(one-time purchase)" after "Outlook 2013 on Windows" as well.

Reply:
No, it is not.